spec/microscope/scope/date_scope_spec.rb

```diff
@@ -84,7 +84,7 @@
       before do
         @event1 = Event.create(started_on: stubbed_date)
         @event2 = Event.create(started_on: stubbed_date + 1.day)
-        Event.create(started_on: 1.month.ago)
```

Comment:
This test was failing because the third event needed to be before the stubbed `today` date